### PR TITLE
All the perf scripts used by LHCb before migrating to the DFC

### PR DESCRIPTION
**Pull request by chaen**

### CODE REVIEW

File: Performance/DFCPerformance/extractResult.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# The name of the server where the DFC service is running
+machineName='volhcb38'
```
Comment (fstagni) [inline Performance/DFCPerformance/extractResult.sh]: Mmmm...

Comment (chaen) [inline Performance/DFCPerformance/extractResult.sh]: laziness to change :p I can make it another argument...

**Comment by fstagni** [comment]
Out of all the tests in Performance/DFCPerformance, what
- needs just a DB?
- needs a service + the DB?
- and what needs an empty DB?

- What should be run in Jenkins?
- What should be run during the system (certification) phase?
- What should NOT be run "systematically"?

**Comment by chaen** [comment]
Nothing should be run systematically, it is *not* to be integrated to Jenkins. I think the README is quite clear, this is just to test if you can sustain your target load.

**Comment by chaen** [comment]
In fact, to be even more precise, we (LHCb) are probably never going to use this anymore :-)

**Pull request merged**